.github/workflows/release.yaml

```diff
@@ -262,11 +262,20 @@ jobs:
           echo "Deleting branch $branch_name"
         done
 
-    - name: Publish Release Event
+    - name: Publish Release Event to Website
       if: inputs.release_candidate == false
       uses: peter-evans/repository-dispatch@v3
       with:
         token: ${{ steps.generate_token.outputs.token }}
         repository: open-component-model/ocm-website
         event-type: ocm-cli-release
         client-payload: '{"tag": "${{ env.RELEASE_VERSION }}"}'
+
+    - name: Publish Release Event for AUR
+      if: inputs.release_candidate == false
```

Comment:
Do i get this correctly: If it is not a release_candidate, then we want to publish it for AUR?

Reply:
yes, if it's not a RC, then it's a normal release, which we want to publish